nodes/nomos-node-api/src/http/info.rs

```diff
@@ -0,0 +1,71 @@
+use std::{fmt::Debug, hash::Hash};
```

Comment:
I would probably split APIs per service instead (i.e. consensus and lip2p in different modules)